src/components/heading/heading.component.js

```diff
@@ -185,32 +187,31 @@ class Heading extends React.Component {
 
     return (
       <StyledHeading
-        className={this.classes}
         divider={this.props.divider}
         subheader={this.props.subheader}
         {...tagComponent("heading", this.props)}
       >
-        <div className="carbon-heading__header">
+        <StyledHeader>
           {this.back}
 
-          <div className="carbon-heading__headers">
-            <div className="carbon-heading__main-header">
-              <h1
-                className="carbon-heading__title"
+          <StyledHeaders back={this.props.backLink}>
+            <Box>
```

Comment:
Does this need to be a `Box`? Could probably just be a div saying as we're not using any `Box` props

Reply:
Box replaced with div